src/Organic/helpers.php

```diff
@@ -25,3 +27,40 @@ function is_valid_uuid( string $uuid ) {
         ? preg_match( '/^[a-f\d]{8}(-[a-f\d]{4}){4}[a-f\d]{8}$/i', $uuid ) === 1
         : false;
 }
+
+/**
+ * Copy of the built-in WordPress function `get_edit_post_link`.
+ * Retrieves the edit post link for post without a permissions check.
+ *
+ * @param int|WP_Post $id      Optional. Post ID or post object. Default is the global `$post`.
+ * @param string      $context Optional. How to output the '&' character. Default '&amp;'.
+ * @return string|null|void    The edit post link for the given post. Null if the post type does not
+ *                             exist or does not allow an editing UI.
+ */
+function get_edit_post_link( $id = 0, string $context = 'display' ) {
```

Comment:
Ah, so `editUrl` was required but we couldn't get it because of permissions issue. Should we explicitly import this function in `OrganicSdk.php` just to make it clear we are using our custom version?

Reply:
Meh. Really it would just be redundant because we're in the same namespace. That said, for an improved developer experience, I'll make it explicit. Not an import, but a fully qualified function call so there won't be any _human_ ambiguity.

Reply:
Even better!